tests/testcases.py

```diff
@@ -118,24 +124,28 @@ def stdout(self):
         clear_stdout was called."""
         return sys.stdout.getvalue()[self.stdout_start:].strip()
 
-    @patch('dusty.commands.utils.exec_docker')
+    def exec_docker_patch(self, *args):
+        args = ['/usr/local/bin/docker'] + [a for a in args]
```

Comment:
you're assuming a binary location here which is basically never okay. use `which` to figure it out.